ui/apps/auth-app/webpack.config.js

```diff
@@ -7,6 +7,6 @@ module.exports = Object.assign(baseConfig, {
   context: path.resolve(__dirname),
   output: Object.assign(baseConfig.output, {
     path: path.resolve(__dirname, 'dist'),
-    publicPath: '/plugins/search/',
+    publicPath: '/apps/auth-app/',
```

Comment:
I think this one should target `search-app`?

Reply:
Think this is correct, it is inside the auth-app folder? Github is showing the change weird though.